src/tasks/publish/createIgnorePatternFromFiles.ts

```diff
@@ -11,7 +10,7 @@ export default function createIgnorePatternFromFiles(
 ): string[] {
   const ignorePatterns: string[] = []
   for (const filename of ['.ipfsignore', '.gitignore']) {
-    const data = readFileIfExists(path.join(rootPath, filename))
```

Comment:
This bit was not necessary. `path.join` is handled it by `readFileIfEists`.

Reply:
Wondering on this—maybe `readFileIfExists()` should just take a single argument as the path (see discussion in #79)?